File: app/src/main/java/com/justcode/hxl/viewutil/自定义View/绘制1_1/饼图/interfaces/PieTextVisibleFilter.java
package com.justcode.hxl.viewutil.自定义View.绘制1_1.饼图.interfaces;


import com.justcode.hxl.viewutil.自定义View.绘制1_1.饼图.entry.PieEntry;

public interface PieTextVisibleFilter<T extends PieEntry> {
    boolean isShowText(T pieEntry);
}

File: app/src/main/java/com/justcode/hxl/viewutil/自定义View/绘制1_1/饼图/interface/PieTextVisibleFilter.kt
package com.justcode.hxl.viewutil.自定义View.绘制1_1.饼图.`interface`

import com.justcode.hxl.viewutil.自定义View.绘制1_1.饼图.entry.PieEntry

interface PieTextVisibleFilter<T : PieEntry> {
    fun isShowText(pieEntry: T): Boolean
}